go.mod

```diff
@@ -73,30 +73,34 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20210817132244-67c28690af52
 	github.com/openshift/cluster-api => github.com/openshift/cluster-api v0.0.0-20191129101638-b09907ac6668
 	github.com/securego/gosec => github.com/securego/gosec v0.0.0-20190709033609-4b59c948083c
-	k8s.io/api => k8s.io/api v0.22.0-rc.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.0-rc.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.22.0-rc.0
-	k8s.io/apiserver => k8s.io/apiserver v0.22.0-rc.0
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.22.0-rc.0
-	k8s.io/client-go => k8s.io/client-go v0.22.0-rc.0
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.22.0-rc.0
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.22.0-rc.0
-	k8s.io/code-generator => k8s.io/code-generator v0.22.0-rc.0
-	k8s.io/component-base => k8s.io/component-base v0.22.0-rc.0
-	k8s.io/component-helpers => k8s.io/component-helpers v0.20.0-alpha.2.0.20210708095128-55a3896515e9
-	k8s.io/controller-manager => k8s.io/controller-manager v0.20.0-alpha.1.0.20210712075914-83508d18fce2
-	k8s.io/cri-api => k8s.io/cri-api v0.22.0-rc.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.22.0-rc.0
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.22.0-rc.0
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.22.0-rc.0
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.22.0-rc.0
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.22.0-rc.0
-	k8s.io/kubectl => k8s.io/kubectl v0.22.0-rc.0
-	k8s.io/kubelet => k8s.io/kubelet v0.22.0-rc.0
-	k8s.io/kubernetes => k8s.io/kubernetes v1.22.0-rc.0
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.22.0-rc.0
-	k8s.io/metrics => k8s.io/metrics v0.22.0-rc.0
-	k8s.io/mount-utils => k8s.io/mount-utils v0.23.0-alpha.0
```

Comment:
@umohnani8 Do we need to keep this version or we can keep it consistent with rest of the replace for kube deps i.e v0.22.1